torch/utils/triton.py

```diff
@@ -0,0 +1,13 @@
+import functools
```

Comment:
this file is copy pasta from inductor utils. I cannot use that since it creates circular dependancy. If this looks like, I can have inductor also use this util. Although there appears two separate implementation for get_device_capability

Reply:
let's move things rather than copying them.

I believe the linter will complain unless the filename starts with `_` (indicating it is not a public API).

Reply:
Okay, I will create a new PR under this one that moves everything to this new API. I was worried about the two different implementations of get_device_capability